2-factor-authentication/README.md

```diff
@@ -5,6 +5,7 @@ It shows you how you can automate a Cross-Application + Cross-Device workflow.
 
 ## Prerequisites
 
+> * Start the ui controller for the desktop with the port `7007` and the android device with the port `6769`.
```

Comment:
Doesn't it make the most sense to put that at the end of the prerequisites list as there are a lot of prequisites for that in the list?

Reply:
i change it to be the last one